src/script/user/UserMapper.ts

```diff
@@ -146,7 +135,7 @@ export class UserMapper {
       if (this.serverTimeHandler.timeOffset() !== undefined) {
         setAdjustedTimestamp();
       } else {
-        this.serverTimeHandler.timeOffset.subscribe_once(setAdjustedTimestamp);
+        (this.serverTimeHandler.timeOffset as any).subscribe_once(setAdjustedTimestamp);
```

Comment:
`subscribe_once` usually doesn't exist in knockout, and I couldn't find a simple solution to add a type to knockout that reflects adding a additional function.

Reply:
Yeah that's a custom function that we add in the `commonBindings` file. 
There should be a way in TS to define those but I am no expert with this.
I am fine with the `any` here.

Reply:
Maybe a cleaner solution would be to make the `subscribe_once` function a raw function (not one that is dynamically added to the prototype). 

Something like `subscribe_once(observable, subscribeFn)` This way we import the file everywhere we need it (instead of relying on another file to have imported it before) and we won't have to tweak typescript.

Reply:
I'll add a comment so we can do this later.